Source/Scene/processPbrMaterials.js

```diff
@@ -91,7 +91,16 @@ define([
         var texCoord;
         if (defined(generatedMaterialValues[textureName + 'Offset'])) {
             var textureIndex = generatedMaterialValues[textureName].index;
-            var sampler = gltf.samplers[gltf.textures[textureIndex].sampler];
+            var samplerSelector = gltf.textures[textureIndex].sampler;
```

Comment:
We would normally call this `samplerIndex` or `sampleName` depending on if we're indexing into an array or object.